diagnostics-extension/src/services/telemetry.ts

```diff
@@ -7,99 +7,140 @@
  */
 
 import {
+  AudioInfo,
   BatteryInfo,
-  BlockDeviceInfo,
+  NonRemovableBlockDeviceInfoResponse,
   CpuInfo,
+  DisplayInfo,
+  MarketingInfo,
   MemoryInfo,
+  InternetConnectivityInfo,
   OemData,
-  StatefulPartitionInfo,
+  OsVersionInfo,
+  UsbBusDevices,
   VpdInfo,
+  StatefulPartitionInfo,
+  TpmInfo,
 } from '@common/dpsl';
-import * as fakeData from './fake_telemetry.data';
+import * as fake_telemetry from './fake_telemetry.data';
```

Comment:
go/tsjs-style#naming-rules-by-identifier-type

I think this is "module alias", so use lowerCamelCase.

Rename to fakeTelemetry

Reply:
Done.